packages/annotorious-openseadragon/vite.config.js

```diff
@@ -28,15 +24,18 @@ export default defineConfig({
     },
     rollupOptions: {
       external: [
-        ...Object.keys(packageJson.peerDependencies)
+        ...Object.keys(packageJson.peerDependencies),
+        '@annotorious/core',
+        '@annotorious/annotorious'
       ],
       output: {
         assetFileNames: 'annotorious-openseadragon.[ext]',
         globals: {
+          '@annotorious/core': 'AnnotoriousCore',
           '@annotorious/annotorious': 'Annotorious',
```

Comment:
@rsimon, could you provide a bit more info about the `umd` consumers and where the global `Annotorious` variable is set? Is it possible to add the `AnnotoriousCore` there as well?

Reply:
I think this might not be needed and/or a mistake by me. If the everything gets bundled cleanly into the main bundle (=global AnnotoriousOSD), then this can be skipped. OpenSeadragon should the only necessary global.